### DISCUSSION TITLE
sanctuaryradio.com disconnects on song change.

### DISCUSSION THREAD

**Discussion by ghost** [MusicPlayerDaemon/MPD#2169] [Q&A] [answered]
I don't know why this is happening, but here's what goes on.

The stream connects
The song ends and either a bumper is played or it goes directly into the next song
That song plays until the info for that song is received (usually less than a second)
The player just stops?

I can hit enter and connect to the streams again.  This is on the HQ main and 80s channels for sanctuaryradio.com  I haven't tried the lower quality ones but I suspect it wouldn't really make a difference.

My best guess is that it's some malformed tag or info put into the stream's data.  I doubt it's a buffering issue as I have good internet and my computer has an ungodly amount of RAM.

I've also tested the stream in Audacious and it works fine in that.  Also it's just that station as detroitindustrial.org works just fine and also updates the song information on track change.

Now, assuming that Rob from Sanctuary Radio doesn't care to fix the issue, how might I go about preventing disconnection or enabling reconnection after that on my end?

**Comment by MaxKellermann** [comment]
Seriously. https://www.musicpd.org/help/
"Your collaboration is necessary to solve your problem. We can’t help you if you don’t follow the instructions above."

**Comment by ghost** [comment, marked as answer]
So, finally figured out I had the log disabled.  Saw "pipewire_output: Decoder is too slow; playing silence to avoid xrun" and then found this discussion: https://github.com/MusicPlayerDaemon/MPD/issues/1771

Adding in the following fixed the disconnection issue
```decoder {
    plugin "oggflac"
    enabled "no"
}```

However as mentioned in that thread, now the track name doesn't seem to update on track change anymore, where as before it would update and just disconnect.

Also I'm not sure why it was disconnecting at all in the first place.  It should have just played silence according to that bug report.
